test-suite/ide/debug_ltac.fake

```diff
@@ -1,2 +1,3 @@
+ADD { Definition foo := 0. }
```

Comment:
Why this change?

Reply:
As written in the commit message:

> We did modify `fake_ide` as not to add a directory with the empty
`Prefix`, which was not used. This exposed some bugs in the
implementation of the document model, which relied on having an
initial sentence; we have workarounded them just by adding a dummy one
in the two relevant cases.

A better solution would be welcome but it is not trivial.

Reply:
It seems like `.fake` files support comments (with `#`). Would you mind adding one explaining this so that it is clear why it is like that in the future?

Reply:
I've used a `Comments` command.